Builders/src/main/scala/Builders.scala

```diff
@@ -0,0 +1,266 @@
+package com.gu.nitf.model.builders
+
+import java.net.URI
+import java.time.{LocalDate, LocalDateTime}
+
+import com.gu.nitf.model._
+import com.gu.nitf.scalaxb._
+import com.gu.scalaxb.{dataRecord => dr}
```

Comment:
That is a bit confusing, is this binding conflicting with another?

Reply:
No, it isn't. It's just shorter and keeps the code, IMHO, more elegant, e.g. [on line 44](#diff-f9327d0c18bd53f7a25d0e413811d63cR42). Do you think it's better without the alias?

Reply:
Yeah I think for a dozen instances it is worth paying the price of slightly less elegant code. If we use a library as a foundation, we should try to reuse existing knowledge as much as possible. The exception would be when they clearly failed to give things a meaningful name, like the `TypeType2` thing 🤔 🤕